assignment5.text

```diff
@@ -0,0 +1,11 @@
+1. call it by typing fxn(a, b, c) where a, b, and c would be constants or variables inputted by the user. It could look like this...
+
+void main() {
+	printf("Enter a number: \n");
+	scanf("%d", a);
+... and then two more lines with b being a float and c being an int.
+	}
+
+2. Iteration is when a loop runs a certain amount of time in order for something to happen while recursion is a function calling itself that updates a counting variable but not changing the variable you want to change at the end. Recursion is not ideal because it takes more memory.
```

Comment:
right, complexity is key! and now that we're learning about memory management and data structures, you can further understand this by understanding that each time a function is called, a stack is initialized. So having a recursive function can be very memory ineffective.
 Add a line note